openhands/server/routes/git.py

```diff
@@ -39,10 +41,7 @@ async def get_user_repositories(
         )
 
         try:
-            repos: list[Repository] = await client.get_repositories(
-                sort, server_config.app_mode
-            )
-            return repos
+            return await client.get_repositories(sort, server_config.app_mode)
```

Comment:
```suggestion
            return await client.get_repositories(sort, server_config.APP_MODE)
```

I think the pydantic model that is introduced uses uppercase now, there may be more instances where we need to replace `app_mode` references with `APP_MODE`